src/lib/score/graphics/widgets/QGraphicsMultiSliderXY.cpp

```diff
@@ -0,0 +1,164 @@
+#include <score/graphics/DefaultGraphicsSliderImpl.hpp>
+#include <score/graphics/widgets/QGraphicsMultiSliderXY.hpp>
+#include <score/model/Skin.hpp>
+#include <score/serialization/StringConstants.hpp>
+#include <score/tools/Debug.hpp>
+
+#include <ossia/detail/math.hpp>
+#include <ossia/network/domain/domain_functions.hpp>
+#include <ossia/network/value/value_conversion.hpp>
+
+#include <QGraphicsSceneMouseEvent>
+#include <QPainter>
+
+#include <wobjectimpl.h>
+W_OBJECT_IMPL(score::QGraphicsMultiSliderXY);
+
+namespace score
+{
+score::QGraphicsMultiSliderXY::QGraphicsMultiSliderXY(QGraphicsItem* parent)
+{
+  auto& skin = score::Skin::instance();
+  setCursor(skin.CursorPointingHand);
+  this->setAcceptedMouseButtons(Qt::LeftButton | Qt::RightButton);
+}
+
+void score::QGraphicsMultiSliderXY::setPoint(const QPointF& r)
+{
+  SCORE_TODO;
+}
+
+void score::QGraphicsMultiSliderXY::paint(
+    QPainter* painter, const QStyleOptionGraphicsItem* option, QWidget* widget)
+{
+
+  tab = m_value.get<std::vector<ossia::value>>();
+
+
+  // Draw the background
+  painter->setBrush(QColor(qRgba(10, 10, 10, 255)));
```

Comment:
Instead of hardcoding colors, could you use some from the skin ? 
see examples here: 

https://github.com/ossia/score/blob/master/src/lib/score/graphics/widgets/QGraphicsRangeSlider.cpp#L150-L155